README.adoc

```diff
@@ -209,6 +206,11 @@ https://www.lucidchart.com/documents/edit/6293d791-8f70-47bb-8136-438128c27129/0
 
```

Comment:
The lucid chart link is only available internally. Should we remove it as the image shows what's in the chart anyway?

Reply:
👍 Done